contrib/test/integration/build/kubernetes.yml

```diff
@@ -5,7 +5,7 @@
     repo: "https://github.com/runcom/kubernetes.git"
     dest: "{{ ansible_env.GOPATH }}/src/k8s.io/kubernetes"
     # based on kube v1.9.0-alpha.2, update as needed
-    version: "cri-o-patched-1.9"
+    version: "{{ version }}"
     force: "{{ force_clone | default(False) | bool}}"
```

Comment:
Arguably both ``version`` and ``force_clone`` are fairly generic names.  Ansible's variable handling is...well...funky, as there's only a single namespace.  Consider if these names will _ever_ need to be used in a different context (e.g. runc version, or cri-o version).  If so, best to make them more specific to avoid future headaches.  I'd suggest:

``k8s_git_version`` and ``k8s_git_force_clone`` (or something like that).

Much easier to fix this now than later 😀

Reply:
Consider setting ``"cri-o-node-e2e-patched-logs"`` as the default, otherwise forgetting to specify the variable will cause an error.  This also lets you omit ``main.yml lines 24-25``.  

e.g. ``version: '{{ k8s_git_version | default("cri-o-node-e2e-patched-logs") }}'``

Reply:
If the branch is not set correctly we might as well fail then trying to make progress with an incorrect branch. I have modified the variable name. Thanks!